scripts/install_script_ubuntu.sh

```diff
@@ -169,13 +170,16 @@ sudo apt-get install -y \
 apt-get --force-yes --yes install python-dev 2>/dev/null
 apt-get --force-yes --yes install libpython-dev 2>/dev/null
 
-run_cmd_confirm wget --quiet https://bootstrap.pypa.io/get-pip.py
-run_cmd_confirm python get-pip.py
-run_cmd_confirm pip install pip --upgrade
+run_cmd_confirm curl -s https://bootstrap.pypa.io/get-pip.py | python
```

Comment:
I was mostly doing it for cleanliness, but I can revert this and add get-pip.py to .gitignore to achieve the same thing.